manifests/generated/rbac-kubevirt.authorization.k8s.yaml.in

```diff
@@ -159,6 +159,26 @@ metadata:
   name: kubevirt-controller
   namespace: {{.Namespace}}
 ---
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: kubevirt-controller-psp
```

Comment:
I know this PSP is used by virt-controller, but it's really reflecting the privileges virt-controller needs to create virt-launcher pods.  Should we call it the virt-launcher PSP instead to reflect that? I know it's all a bit convoluted